test-image/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM agpipeline/base-image:1.0
```

Comment:
It would be better to build the original docker image from the main source tree, and then build the test docker image by copying the needed files into a new instance.  Something like `docker build -f ../Dockerfile -t base-image:latest ../` and then `docker build -t test-image:latest test-image` using a dockerfile starting with `FROM base-image:latest`. See me if you have questions on this. Also, feel free to create a ticket to do this (instead of doing this now)

Reply:
After all the suffering I went through to finally get this working you pull me back in.
In all seriousness though, that sounds fine. I'll set up a ticket and maybe get to it later this week if not early next week since I'm pretty sure it'll cause problems somewhere.